feg/gateway/services/n7_proxy/servicers/multiple_session_controller.go

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright 2021 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
This looks a copy of session_proxy. Why is multiple session controller needed at all?

Reply:
This is for having feature parity with Gx where multiple PCRF can be used. Similarly in n7_proxy multiple session controllers is an abstraction used to connect to multiple PCFs. Do you thing the FEG will connect to only one PCF?